python/paddle/v2/fluid/tests/book_distribute/notest_recognize_digits_conv_dist.py

```diff
@@ -58,14 +58,19 @@
 current_endpoint = os.getenv("SERVER_ENDPOINT")  # current pserver endpoint
 training_role = os.getenv("TRAINING_ROLE",
                           "TRAINER")  # get the training role: trainer/pserver
+if not current_endpoint:
+    print("need env SERVER_ENDPOINT")
+    exit(1)
+
 t = fluid.DistributeTranspiler()
 t.transpile(
-    optimize_ops, params_grads, pservers=pserver_endpoints, trainers=trainers)
+    optimize_ops,
+    params_grads,
+    0,
```

Comment:
Need to get trainer_id from ENV or command-line arguments.
I will fix it in a new PR.